CHANGELOG.md

```diff
@@ -5,78 +5,508 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+- Unfork `cpuid-bool`, not used anymore
+- Unfork `grpcio` and bump from 0.6 to 0.10.3. ([#1592], [#1717], [#1814])
+- Unfork `prost` from bump from 0.8.0 to 0.10.3 ([#898], [#1109], [#1728], [#1809], [#1809], [#1806], [#1805], [#1808], [#1808], [#1807], [#1930], [#1927], [#1926], [#1929])
+- Update `cmake` fork to git-5f89f90ee5d7789832963bffdb2dcb5939e6199c
+- Update `curve25519-dalek` fork from 4.0.0-pre.0 to 4.0.0-pre.2
+- Update `ed25519-dalek` fork to support new rust nightlies
+- Update `mbedtls`, `mbedtls-sys` forks to support newer rust nightlies, use newer `spin`
+- Update `x25519-dalek` fork to support newer rust nightlies
+
+- Bump `aead` from 0.3.2 to 0.4.3 ([#1389])
+- Bump `aes-gcm` from 0.9.2 to 0.9.4
+- Bump `aes` from 0.7.4 to 0.7.5
+- Bump `anyhow` from 1.0.39 to 1.0.57 ([#1013], [#1146], [#1265], [#1341], [#1529], [#1578], [#1837])
+- Bump `arrayvec` from 0.5.2 to 0.7.1 ([#980])
+- Bump `assert_cmd` from 2.0.2 to 2.0.4 ([#1314])
+- Bump `backtrace` from 0.3.55 to 0.3.65 ([#982], [#1143], [#1392], [#1817], [#1817])
+- Bump `base64` from 0.12.3 to 0.13.0
+- Bump `bincode` from 1.3.1 to 1.3.3 ([#1056])
+- Bump `bindgen` from 0.51.1 to 0.59.2
+- Bump `bitflags` from 1.2.1 to 1.3.2 ([#1016])
+- Bump `blake2` from 0.9.1 to 0.10.4 ([#1520])
+- Bump `bs58` from 0.3.1 to 0.4.0 ([#948])
+- Bump `cargo-emit` from 0.1.1 to 0.2.1 ([#1045], [#990], [#1000], [#937], [#968])
+- Bump `cargo_metadata` from 0.9.1 to 0.14.2 ([#949], [#1135], [#1502])
+- Bump `cbindgen` from 0.14.3 to 0.23.0 ([#1020], [#1702], [#1824], [#1824], [#1836])
+- Bump `cc` from 1.0.66 to 1.0.73 ([#919], [#920], [#985], [#983], [#1099], [#1094], [#1097], [#1095], [#1096], [#1168], [#1165], [#1166], [#1164], [#1167], [#1501], [#1498], [#1497], [#1499], [#1500])
+- Bump `cfg-if` from 0.1.10 to 1.0.0
+- Bump `chrono` from 0.4.11 to 0.4.19 ([#959])
+- Bump `clap` from 3.1.6 to 3.1.18 ([#1762], [#1825], [#1847], [#1904], [#1957])
+- Bump `cookie` from 0.14.3 to 0.16.0 ([#1034], [#1271])
+- Bump `crc` from 1.8.1 to 2.0.0 ([#1018], [#1138], [#1857])
+- Bump `criterion` from 0.3.2 to 0.3.5 ([#1059])
+- Bump `crossbeam-channel` from 0.5.0 to 0.5.4 ([#1039], [#1313], [#1678])
+- Bump `diesel-derive-enum` from 1.1.1 to 1.1.2 ([#1311])
+- Bump `diesel` from 1.4.7 to 1.4.8 ([#1061])
+- Bump `digest` from 0.9.0 to 0.10.1
+- Bump `dirs` from 2.0.2 to 4.0.0 ([#1071])
+- Bump `displaydoc` from 0.2.0 to 0.2.3 ([#936], [#933], [#995])
+- Bump `ed25519` from 1.0.1 to 1.5.0 ([#1179], [#1679], [#1950])
+- Bump `futures` from 0.3.8 to 0.3.21 ([#1017], [#1262], [#1458])
+- Bump `generic-array` from 0.14.4 to 0.14.5 ([#1315])
+- Bump `getrandom` from 0.1.13, 0.2.2 to 0.2.6 ([#986], [#1052], [#1031], [#1310], [#1387], [#1532], [#1531], [#1714], [#1712])
+- Bump `ghash` from 0.4.2 to 0.4.4
+- Bump `hashbrown` from 0.6.3 to 0.12.1 ([#899], [#1915])
+- Bump `hdkf` from 0.9.0 to 0.12.3
+- Bump `hex` from 0.4.2 to 0.4.3 ([#1006], [#923], [#909], [#975], [#913])
```

Comment:
Typo in `hkdf`
```suggestion
- Bump `hex` from 0.4.2 to 0.4.3 ([#1006], [#923], [#909], [#975], [#913])
- Bump `hdkf` from 0.9.0 to 0.12.3 (#1345)
```

Reply:
We really do need to put those in braces so they link properly with the link notation at the end of this section (also there's a typo in `hkdf` :-)